projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-transient.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * REST API endpoint for editing Jetpack Transients.
+ *
+ * @package automattic/jetpack
+ * @since 9.7.0
+ */
+
+/**
+ * Edit and delete transients.
+ *
+ * @since 9.7.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Transient extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'transients';
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 *
+	 * /sites/$site/transients/$name/delete
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<name>\w{1,172})/delete/',
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'delete_transient' ),
+					'permission_callback' => 'is_user_logged_in',
+				),
```

Comment:
Let's add the argument description here, something like ...

```php
'name' => array(
	'description' => __( 'The transient name to be deleted.', 'jetpack' ),
	'type'        => 'string',
),
```

Reply:
I [added this](https://github.com/Automattic/jetpack/pull/19632/commits/1cc2f87c10fc78ec6dadb05634b996a02d8c207d), but I didn't find any similar arg descriptions in the other endpoints. 🤔 They're always query args wrapped in an `args` array.

Reply:
oh, right. thanks.